docs/how-to-guides/azure-arc-onboarding.md

```diff
@@ -0,0 +1,223 @@
+
+```bash
+az extension add --name k8s-configuration
+```
+
+#### Create a configuration
+
+```bash
+az k8s-configuration create \
+  --name cluster-config \
+  --cluster-name mercury \
+  --resource-group AzureArcTest \
+  --operator-instance-name cluster-config \
+  --operator-namespace cluster-config \
+  --repository-url https://github.com/ipochi/azure-arc-nginx-demo \
+  --scope cluster \
+  --cluster-type connectedClusters \
+  --operator-params='--git-branch=main'
+```
+
+- `name` - name of the Kubernetes configuration you want.
+- `cluster-name` - name of the cluster provided in `azure-arc-onboarding` component configuration.
+- `resource-group` - Azure Resource group name provided in the `azure-arc-onboarding` component configuration.
+- `operator-instance-name` - name of the Operation instance.
+- `repository-url` - name of the Git repository.
+- `scope` - scope of the operator. Accepted values: `cluster` or `namespace`.
+- `cluster-type` - Arc cluster type. Accepted values: `connectedClusters` or `managedClusters`.
+- `operator-params` - Additional parameters to pass to the Operator. In this case, change default git branch to `main`.
+
+This creates two Deployments in the provided namespace `cluster-config`, corresponding to the GitOps agent
+that Azure Arc uses:
+
+```bash
+kubectl get deployments -n cluster-config
+NAME                         READY   UP-TO-DATE   AVAILABLE   AGE
+cluster-config               1/1     1            1           103m
+memcached-cluster-config     1/1     1            1           103m
+
+kubectl get pods -n cluster-config
+NAME                                          READY   STATUS    RESTARTS   AGE
+cluster-config-5f68dd9bd5-xsmc9               1/1     Running   0          5d18h
+memcached-cluster-config-857d47969f-v2g8q     1/1     Running   0          5d18h
+```
+
+Once the `cluster-config` pods are in `Running` state, you'll notice that a new resources are created in the
+namespace `demo`. The manifests for these resources are pulled from the provided Git repository.
+
+```bash
+k get all -n demo
+NAME                           READY   STATUS    RESTARTS   AGE
+pod/my-nginx-5bcdc5784-fb87k   1/1     Running   0          105m
+pod/my-nginx-5bcdc5784-g2gt2   1/1     Running   0          105m
+
+NAME               TYPE       CLUSTER-IP     EXTERNAL-IP   PORT(S)          AGE
+service/my-nginx   NodePort   10.3.250.153   <none>        8080:32500/TCP   105m
+
+NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/my-nginx   2/2     2            2           105m
+
+NAME                                  DESIRED   CURRENT   READY   AGE
+replicaset.apps/my-nginx-5bcdc5784    2         2         2       105m
+replicaset.apps/my-nginx-5f4dfbff8c   0         0         0       105m
+```
+
+If you used your own Git repository url , try making the changes and committing to your repository. You'll
+notice the changes propagate instantly without the need for any manual intervention.
```

Comment:
This looks like very neat feature of Arc. Also the guide describing it is nicely written, I like it :)

Reply:
our how-to guide is nicely written or the Azure documentation regarding this feature ?

Reply:
Your guide of course :)